src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1579,6 +1616,119 @@ public void TestConnectionStringEnvVarHasHighestPrecedence()
+        /// {
+        ///     "nested-mutations": {}
+        /// }
+        ///
+        /// 3. Create field within Nested Mutation section is null.
+        /// {
+        ///     "nested-mutations": {
+        ///         "create": null
+        ///     }
+        /// }
+        ///
+        /// 4. Create field within Nested Mutation section is empty.
+        /// {
+        ///     "nested-mutations": {
+        ///         "create": {}
+        ///     }
+        /// }
+        /// 
+        /// For all the above mentioned scenarios, the expected value for NestedMutationOptions field in null.
+        /// </summary>
+        /// <param name="baseConfig">Base Config Json string.</param>
+        [DataTestMethod]
+        [DataRow(TestHelper.BASE_CONFIG_NULL_NESTED_MUTATIONS_FIELD, DisplayName = "Validate successful deserialization when nested mutation section is null")]
+        [DataRow(TestHelper.BASE_CONFIG_EMPTY_NESTED_MUTATIONS_FIELD, DisplayName = "Validate successful deserialization when nested mutation section is empty")]
+        [DataRow(TestHelper.BASE_CONFIG_NULL_NESTED_CREATE_FIELD, DisplayName = "Validate successful deserialization when create field within nested mutation section is null")]
+        [DataRow(TestHelper.BASE_CONFIG_EMPTY_NESTED_CREATE_FIELD, DisplayName = "Validate successful deserialization when create field within nested mutation section is empty")]
+        public void ValidateDeserializationOfConfigWithNullOrEmptyInvalidNestedMutationSection(string baseConfig)
+        {
+            string configJson = TestHelper.AddPropertiesToJson(TestHelper.BASE_CONFIG, BOOK_ENTITY_JSON);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(configJson, out RuntimeConfig deserializedConfig));
+            Assert.IsNotNull(deserializedConfig.Runtime);
+            Assert.IsNotNull(deserializedConfig.Runtime.GraphQL);
+            Assert.IsNull(deserializedConfig.Runtime.GraphQL.NestedMutationOptions);
+        }
+
+        /// <summary>
+        /// Sanity check to validate that DAB engine works fine when used with a config file without the nested 
+        /// mutations feature flag section.
+        /// The runtime graphql section of the config file used looks like this: 
+        ///
+        /// "graphql": {
+        ///    "path": "/graphql",
+        ///    "allow-introspection": true
+        ///  }
+        /// 
+        /// Without the nested mutations feature flag section, DAB engine should be able to 
+        ///  1. Successfully deserialize the config file without nested mutation section.
+        ///  2. Process REST and GraphQL API requests.
+        /// 
+        /// </summary>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public async Task SanityTestForRestAndGQLRequestsWithoutNestedMutationFeatureFlagSection()
+        {
+            // The config file is constructed by merging the hard-coded json strings to mimic the scenario where config file is
```

Comment:
```suggestion
            // The configuration file is constructed by merging hard-coded JSON strings to simulate the scenario where users manually edit the configuration file (instead of using the CLI).
```